spec/application/application.js

```diff
@@ -0,0 +1,61 @@
+const Validator = require('is-my-ssb-valid')
+const definitions = require('ssb-schema-definitions')()
+const schema = {
+  $schema: 'http://json-schema.org/schema#',
+  type: 'object',
+  required: ['type', 'version', 'tangles'],
+  properties: {
+    type: {
+      type: 'string',
+      pattern: '^group/application$'
+    },
+    version: {
+      type: 'string',
+      pattern: '^v1$'
+    },
+    groupId: {
+      $ref: '#/definitions/cloakedMessageId'
+    },
+    addMember: {
```

Comment:
addMember is not used, remove this 
I can do this

Reply:
Oh i see this is used in replies, my bad.
I might split out the root and update schemas